contracts/finance/andromeda-vesting/src/testing/tests.rs

```diff
@@ -700,7 +695,7 @@ fn test_claim_batch_not_nice_numbers_single_release() {
 
     deps.querier
         .base
-        .update_balance(MOCK_CONTRACT_ADDR, coins(7, "uusd"));
+        .update_balance(MOCK_CONTRACT_ADDR, coins(10, "uusd"));
```

Comment:
There's `not_nice_numbers` in the test's name but we're using the number 10, which is pretty nice.

Reply:
resolved